labellab-flask/api/controllers/issuecontroller.py

```diff
@@ -0,0 +1,185 @@
+                        
+            response = {
+                "success": True,
+                "msg": "Issues found",
+                "body": "Get Issue by ID"
+            }
+            return make_response(jsonify(response)), 200
+        
+        except Exception:
+            response = {
+                "success":False,
+                "msg": "Something went wrong!"
+                }
+            # Return a server error using the HTTP Error Code 500 (Internal
+            # Server Error)
+            return make_response(jsonify(response)), 500
+    
+    @jwt_required
+    def put(self,project_id, issue_id):
+        try:
+                        
+            response = {
+                "success": True,
+                "msg": "Issues updated",
+                "body": "Issue Details upadted"
+            }
+            return make_response(jsonify(response)), 200
+        
+        except Exception:
+            response = {
+                "success":False,
+                "msg": "Something went wrong!"
+                }
+            # Return a server error using the HTTP Error Code 500 (Internal
+            # Server Error)
+            return make_response(jsonify(response)), 500
+    
+    @jwt_required
+    def delete(self,project_id, issue_id):
+        try:
+                        
+            response = {
+                "success": True,
+                "msg": "Issues Deleted",
+                "body": "Issue Details deleted"
+            }
+            return make_response(jsonify(response)), 200
+        
+        except Exception:
+            response = {
+                "success":False,
+                "msg": "Something went wrong!"
+                }
+            # Return a server error using the HTTP Error Code 500 (Internal
+            # Server Error)
+            return make_response(jsonify(response)), 500
+
+class FetchCategoryIssuesView(MethodView):
+
+    """
+    This route fetches all project issue of a particular category
+     URL:- /api/v1/issue/<int:project_id>/category/<string:category>
+    """
+    @jwt_required
+    def get(self,project_id, category):
+        try:
+                        
+            response = {
+                "success": True,
+                "msg": "Issues found",
+                "body": "Get Issue by Category"
+            }
+            return make_response(jsonify(response)), 200
+        
+        except Exception:
+            response = {
+                "success":False,
+                "msg": "Something went wrong!"
+                }
+            # Return a server error using the HTTP Error Code 500 (Internal
+            # Server Error)
+            return make_response(jsonify(response)), 500
+
+class FetchTeamIssuesView(MethodView):
+
+    """
+    This route fetches all project issue of a particular category
+     URL:- /api/v1/issue/<int:project_id>/team/<string:team>
+    """
+    @jwt_required
+    def get(self,project_id, team):
```

Comment:
It should be `/api/v1/issue/<int:project_id>/team/<int:team_id>`. 
Team id is the primary key here, not team.